website/docs/components/input-phone/input-phone-a11y.md

```diff
@@ -4,38 +4,10 @@ a11y: AA
 tabs: Design('input-phone'), A11y('input-phone-a11y'), Example('input-phone-code')
 ---
 
-## What component has
+This component consists of Input and Select elements, so feel free to check their detailed documentation regarding accessibility:
```

Comment:
wouldn't it be better to call "Input and Select" components instead of elements?
and also, is it correct to call InputPhone a component? It looks more like a pattern. We might have reasons not to move it away from other Input* components, but maybe it'll be safer to avoid the word "component" when mentioning it?

Reply:
> wouldn't it be better to call "Input and Select" components instead of elements?
OMG, yes, of course, it's a typo. I'll fix this.

> and also, is it correct to call InputPhone a component? It looks more like a pattern. We might have reasons not to move it away from other Input* components, but maybe it'll be safer to avoid the word "component" when mentioning it?
Good point! 👍 Agree on this so much.